### DISCUSSION TITLE
Сreating a watermark on the canvas with the ability to zoom in or out

### DISCUSSION THREAD

**Discussion by i-prikot** [fabricjs/fabric.js#7169] [Show and tell]
Maybe someone will come in handy. )

http://jsfiddle.net/i_prikot/zec2r7vp/

https://user-images.githubusercontent.com/21138391/123939053-ee9b4200-d9c1-11eb-8480-1ab03b43210b.mp4

**Comment by asturur** [comment]
@i-prikot you are doing nice examples, why you don't try to just add a demo page to the website? is as simple as making a codepen AND elaborate with some english around it, sort of story telling.

look at how is done here
https://raw.githubusercontent.com/fabricjs/fabricjs.com/gh-pages/posts/demos/_posts/2020-2-15-custom-control-render.md

you add a MD file to the folder where this is contained.
you use MD markup or plain html to make your text or insert images.
Then as the example is doing you can define a codepen on the page that will show your live example, that people could thinker with directly on the website.
http://fabricjs.com/custom-control-render

This are precious contributions for us, because they expand and explain better how to use the library.
This is valid for all the 3 examples you submitted

**Reply by i-prikot** [reply]
@asturur, i believe that my technical knowledge of English is not enough to write an example. I would be glad if someone writes according to my examples.

**Reply by asturur** [reply]
Don't think my english is great either.
I'm italian.
If that is your only concern, give it a shot, the value added you would give us with an example for the website is high.

**Reply by i-prikot** [reply]
Ok. I think I can do it at the end of the summer.
